protocols: unit tests are failing

Some unit tests are failing with Golang 1.13

```
-timeout 30s'

=== RUN   TestNewAgentClient

--- FAIL: TestNewAgentClient (1.01s)

	Error Trace:	client_test.go:114

			client_test.go:124

			client_test.go:134

	Error:      	Should be true

	Test:       	TestNewAgentClient

	Messages:   	expect err message: Invalid vsock port	got: parse vsock://100:bar: invalid port ":bar" after host

=== RUN   TestNewAgentClientWithYamux

--- FAIL: TestNewAgentClientWithYamux (1.02s)

	Error Trace:	client_test.go:114

			client_test.go:148

			client_test.go:156

	Error:      	Should be true

	Test:       	TestNewAgentClientWithYamux

	Messages:   	expect err message: Invalid vsock port	got: parse vsock://100:bar: invalid port ":bar" after host

=== RUN   TestParseGrpcHybridVSockAddr

--- PASS: TestParseGrpcHybridVSockAddr (0.00s)

FAIL

coverage: 62.7% of statements
```

--- a/protocols/client/client_test.go
+++ b/protocols/client/client_test.go
@@ -131,7 +131,7 @@ func TestNewAgentClient(t *testing.T) {
 	cliFunc(mockBadSchemeAddr, false, "Invalid scheme:")
 	cliFunc(mockBadVsockScheme, false, "Invalid vsock scheme:")
 	cliFunc(mockVsockBadCid, false, "Invalid vsock cid")
-	cliFunc(mockVsockBadPort, false, "Invalid vsock port")
+	cliFunc(mockVsockBadPort, false, "invalid port")
 	cliFunc(mockFakeVsockAddr, false, "context deadline exceeded")
 
 	// wait mock server to stop
@@ -153,7 +153,7 @@ func TestNewAgentClientWithYamux(t *testing.T) {
 	cliFunc(mockBadSchemeAddr, false, "Invalid scheme:")
 	cliFunc(mockBadVsockScheme, false, "Invalid vsock scheme:")
 	cliFunc(mockVsockBadCid, false, "Invalid vsock cid")
-	cliFunc(mockVsockBadPort, false, "Invalid vsock port")
+	cliFunc(mockVsockBadPort, false, "invalid port")
 	cliFunc(mockFakeVsockAddr, false, "context deadline exceeded")
 
 	// wait mock server to stop

